_content/booklets/awareness-itself_fuang.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Awareness Itself: The Teachings of Ajaan Fuang Jotiko"
+authors:
+  - fuang
+translator: geoff
+external_url: "https://buddho.org/book/awareness-itself/"
+drive_links:
+  - "https://drive.google.com/file/d/1SPeJsd86iGZeruJeNRV7kx5tEaz0wAWI/view?usp=sharing"
+course: thai
```

Comment:
Also include the epub drive_link: https://drive.google.com/file/d/14NssgN2r4iGvJjuLOgZTtz1qNUVY582h/view?usp=drivesdk